public/index.html

```diff
@@ -4,9 +4,11 @@
 <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=5, user-scalable=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Fida - Decentralized Insurance Marketplace" />
+    <link rel="preload" as="image" href="/src/assets/images/founder-hello-card-mobile.png">
```

Comment:
Use media queries here to avoid preloading both images 

see https://medium.com/@francis.john/viewport-and-media-query-based-preloading-87db30ff6d5f

Reply:
@kryptt with adding media to preload link performance dropped from 73 to 61

Reply:
also size of each photo is between 500 - 600kb, I think it's not a big deal if we've got higher performance :D